modules/create-datavolume-from-manifest/build/create-datavolume-from-manifest/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal AS builder
+RUN microdnf install -y golang-1.16.* && microdnf clean all
```

Comment:
can you please bump golang to 1.18?

Reply:
This is only for onboarding purposes. The golang version will be bumped in the PR with the actual changes.